kihuha/iReporter-db
# #162677234 - Fix token generator

#### What does this PR do?
Implements a dynamic way of generating persistent tokens per user

#### Description of the task to be completed
Generate a new token each time a new user logs in with the correct payload

#### How should this be manually tested
Clone the repo and on your terminal, navigate to the root directory of this repository. Type `python run.py` to start the server and using postman, send a login post request to `<localhost>/api/v2/users/login`. The body of the request should contain a valid username and password as shown below. 

```
{
	"username":"<username>",
	"password":"<password>"
}
```

#### Relevant PT stories
https://www.pivotaltracker.com/story/show/162677234
